libraries/ui/src/main/java/androidx/media3/ui/PlayerView.java

```diff
@@ -1138,6 +1138,16 @@ public void setFullscreenButtonClickListener(@Nullable FullscreenButtonClickList
     controller.setOnFullScreenModeChangedListener(componentListener);
   }
 
+  /**
+   * Sets whether the fullscreen button is fullScreenButton or minimalFullScreenButton.
+   *
+   * @param isFullscreen Whether the player is currently in fullscreen.
```

Comment:
Fullscreen state is semantically inverse to the button display.

* When the player is fullscreen -> button should show "exit fullscreen"
* When the player is minimized -> button should show "enter fullscreen"

It might be best to document this method in terms of the icon drawable you will be setting (since you can't really claim to be imperatively setting the player to a fullscreen activity)

* exo_styled_controls_fullscreen_enter
* exo_styled_controls_fullscreen_exit

Reply:
I have rephrased this javadoc too in 2f1b08fa:
Updates whether the controller is in fullscreen, changing its fullscreen icon and reports it to to the listener.

For isFullscreen equals true the icon will be set to ``exo_styled_controls_fullscreen_exit`` or else ``exo_styled_controls_fullscreen_enter``.